src/vma/proto/neighbour.cpp

```diff
@@ -2007,12 +2012,7 @@ int neigh_ib::find_pd()
 {
 	neigh_logdbg("");
 
-	if (m_cma_id->verbs == NULL) {
-		neigh_logdbg("m_cma_id->verbs is NULL");
-		return -1;
-	}
-	ib_ctx_handler* ib_ctx_h = g_p_ib_ctx_handler_collection->get_ib_ctx(
-			m_cma_id->verbs);
+	ib_ctx_handler* ib_ctx_h = g_p_ib_ctx_handler_collection->get_ib_ctx(m_p_dev->get_ifname_link());
```

Comment:
What about ib_ctx_handler_collection::get_ib_ctx(struct ibv_context* p_ibv_context) ? can we remove it too ?